src/main/java/uk/gov/hmcts/reform/professionalapi/domain/ProfessionalUser.java

```diff
@@ -84,5 +84,10 @@ public ProfessionalUser(
         this.emailAddress = emailAddress;
         this.status = status;
         this.organisation = organisation;
+        this.userIdentifier = generateUserIdentifier();
+    }
+
```

Comment:
Ensure that this is not being duplicated in RDCC-119

Reply:
Created new Util class.